tests/tracing_test.go

```diff
@@ -238,3 +378,15 @@ func (m *mockTracer) verifySpans(spanNames ...string) error {
 
 	return nil
 }
+
+func (m *mockTracer) getOrderedSpan() []string {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	c := make([]string, len(m.orderedSpans))
+	copy(c, m.orderedSpans)
```

Comment:
https://pkg.go.dev/slices#Clone?

Reply:
Why clone over copy?

Reply:
`Clone` does the same with a single line, and it's built in the Standard Library. No strong opinion. Your call :)

Reply:
I'll leave this as is for now. Thanks for the info on `clone` though 👍

Reply:
I was there doing another change, so made the clone change too 🙂. Amended in 8ab99ffc5a249f83adc1a44461b4908ae167f63b

Reply:
I've had to revert the clone since some CI pipelines still work with go 1.20 where the slices package isn't available.